src/core_modules/capture-core/components/WidgetEnrollment/EnrollmentDate/EnrollmentDate.component.js

```diff
@@ -0,0 +1,151 @@
+// @flow
```

Comment:
NIT: Can we change the name of this file since we are using it for both enrollment date and incident date? A bit hard since it is the enrollment Widget (so EnrollmentDate makes some sense). EnrollmentWidgetDate maybe? (feel free to come up with something better)